src/main/java/com/suse/saltstack/netapi/calls/modules/Match.java

```diff
@@ -0,0 +1,62 @@
+package com.suse.saltstack.netapi.calls.modules;
+
+import com.google.gson.reflect.TypeToken;
+import com.suse.saltstack.netapi.calls.LocalCall;
+import com.suse.saltstack.netapi.utils.ClientUtils;
+
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+/**
+ * salt.modules.match
+ */
+public class Match {
```

Comment:
Thanks for opening the issue, @lucidd! From my perspective there is no need to add javadoc for the `Match` module class within this patch, especially I wouldn't want to copy it from the salt documentation for the reasons mentioned in #117. For now let's just do it as the other module classes we already have, i.e. keep it as it is.